### PR TITLE
Update link (rooms-deploy.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9533.

### CODE REVIEW

File: Teams/rooms/rooms-deploy.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ You can use [remote PowerShell](rooms-operations.md#remote-management-using-powe
 
 ### Testing
 
-After Teams Rooms has been deployed, you should test it. Check that the capabilities listed in [Microsoft Teams Rooms help](https://support.microsoft.com/en-us/office/microsoft-teams-rooms-help-e667f40e-5aab-40c1-bd68-611fe0002ba2?ui=en-us&rs=en-us&ad=us) are working on the deployed device. We highly recommend that the deployment team verify that Microsoft Teams Rooms is appearing in Teams admin center. It's also important that you make a number of test calls and meetings to check quality. For more information, see this [useful deployment checklist](console.md#microsoft-teams-rooms-deployment-checklist).
+After Teams Rooms has been deployed, you should test it. Check that the capabilities listed in [Microsoft Teams Rooms](https://www.microsoft.com/en-ww/microsoft-teams/microsoft-teams-rooms) are working on the deployed device. We highly recommend that the deployment team verify that Microsoft Teams Rooms is appearing in Teams admin center. It's also important that you make a number of test calls and meetings to check quality. For more information, see this [useful deployment checklist](console.md#microsoft-teams-rooms-deployment-checklist).
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-deploy.md:170]:
```suggestion
After Teams Rooms has been deployed, you should test it. Check that the capabilities listed in [Microsoft Teams Rooms](https://www.microsoft.com/microsoft-teams/microsoft-teams-rooms) are working on the deployed device. We highly recommend that the deployment team verify that Microsoft Teams Rooms is appearing in Teams admin center. It's also important that you make a number of test calls and meetings to check quality. For more information, see this [useful deployment checklist](console.md#microsoft-teams-rooms-deployment-checklist).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**